config/initializers/content_security_policy.rb

```diff
@@ -4,19 +4,19 @@
 # For further information see the following documentation
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
 
-# Rails.application.config.content_security_policy do |policy|
+Rails.application.config.content_security_policy do |policy|
```

Comment:
I'm not sure you want to turn this CSP on. Well I guess I see you are not setting anything. Do you have CSP enabled elsewhere? Normally, you wouldn't need to set this up for dev unless CSP was turned on. I guess it's a good thing if you do.

Reply:
hmm I did this following the guidance under the [Development](https://github.com/rails/webpacker#development) section in the readme:

> Note: You need to allow webpack-dev-server host as an allowed origin for connect-src if you are running your application in a restrict CSP environment (like Rails 5.2+). This can be done in Rails 5.2+ in the CSP initializer config/initializers/content_security_policy.rb with a snippet like this:
> 
>   Rails.application.config.content_security_policy do |policy|
>     policy.connect_src :self, :https, 'http://localhost:3035', 'ws://localhost:3035' if Rails.env.development?
>   end